api/deploy.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -eu
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ -z "${TAG:=}" ]]; then
+    export TAG=dev;
+    echo "Defaulted TAG to '${TAG}'";
+fi
+
+if [[ -z "${OAUTH_CLIENT_ID:=}" ]]; then
+    export OAUTH_CLIENT_ID="863461568634-mjhsbfk81u5pognae6p19jjn5uph5rqn.apps.googleusercontent.com";
```

Comment:
@mservidio  I wouldn't include a valid client id as the default value even if it's non-sensitive. What about a generic *change-me* string?